Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/pages/OptionsMenu.java

```diff
@@ -0,0 +1,66 @@
+package com.automattic.simplenote.smoke.pages;
+
+import com.automattic.simplenote.R;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+public class OptionsMenu extends BasePage {
+
+    private static final Integer POSITION_MENU = 2;
+    private static final String MENU_DESCRIPTION = "More Options";
```

Comment:
Could we use resource IDs for this and other elements in this menu, instead of string literals?

Reply:
As you said resource IDs are better for handling this 👍